Tests/PostmarkClientTests/MailClientTests.swift

```diff
@@ -0,0 +1,27 @@
+import XCTest
+import Mail
+import PostmarkClient
+import HTTP
+import Transport
+import Vapor
+
+class MailClientTests: XCTestCase {
+    static let allTests = [
+        ("testSendEmail", testSendEmail),
+    ]
+    
+    let testServerToken = "POSTMARK_API_TEST"
```

Comment:
Postmark kindly provides this `POSTMARK_API_TESTS` token that makes their backend respond like it's a real `serverToken`, runs all validations and stuff, but doesn't send any actual emails. Very useful.